### PR TITLE
Assert commands results. Remove CA download, machines already have it. Do not install a library through pip.

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by lhellebr** [comment]
Test's setup passed and it failed later. Doesn't seem like this PR's fault. I tested on 6.15 without issues, I'll test on stream and comment the results.

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6190
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout --external-logging
Test Result : ================== 1 failed, 1 warning in 2986.14s (0:49:46) ===================
```

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6204
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout --external-logging
Test Result : ================== 1 failed, 1 warning in 2988.45s (0:49:48) ===================
```

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6212
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout --external-logging
Test Result : ================== 1 passed, 1 warning in 2945.55s (0:49:05) ===================
```

**Review by pondrejk — APPROVED** [review]

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6268
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout --external-logging
Test Result : ================== 1 passed, 4 warnings in 2602.95s (0:43:22) ==================
```

**Comment by lhellebr** [comment]
@SatelliteQE/airgun-tier-1-reviewers

**Comment by lhellebr** [comment]
@SatelliteQE/robottelo-tier-2-reviewers

File: pytest_fixtures/component/satellite_auth.py [hunk 1]
```diff
@@ -282,42 +282,54 @@ def auth_data(request, ad_data, ipa_data):
...
                 --app-name foreman-openidc \
                 --keycloak-server-url {settings.rhsso.host_url} \
                 --keycloak-admin-username "admin" \
                 --keycloak-realm "{settings.rhsso.realm}" \
                 --keycloak-admin-realm master \
                 --keycloak-auth-role root-admin -t openidc -l /users/extlogin --force'
+        ).status
+        == 0
     )
     if is_open('BZ:2113905'):
```
Comment (shweta83) [inline pytest_fixtures/component/satellite_auth.py:315]: This condition can be removed as this BZ is verfied and closed now.

**Review by shweta83 — CHANGES_REQUESTED** [review]

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6646
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout --external-logging
Test Result : ================== 12 warnings, 1 error in 1561.60s (0:26:01) ==================
```

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout

**Comment by lhellebr** [comment]
BZ has been converted to docs, lxml package now should be installed manually according to docs. Updating accordingly.

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6711
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/destructive/test_ldap_authentication.py::test_session_expire_rhsso_idle_timeout --external-logging
Test Result : ================= 1 passed, 17 warnings in 2393.81s (0:39:53) ==================
```

**Review by ogajduse — APPROVED** [review]

**Comment by ogajduse** [comment]
PRT is passing, and Shweta's comment was addressed. Merging.

**Pull request merged**